docs/user-guide/remapping.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Rephrase:</p><p> </p><p>We can view the supported remapping methods by accessing the <code>.remap</code> attribute that is part of a <code>UxDataArray</code> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/807/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>